ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -4,3 +4,14 @@
 
 def cat(tensors, dim=0, *, out=None):
     return ivy.concat(tensors, dim, out=out)
+
+
+def permute(input, dims):
+    return ivy.permute_dims(input, dims)
```

Comment:
I just confirmed that ivy.permute_dims have a out argument just like ivy.concat(), so you need to add out=Non argument to your permute function and put that out in to the ivy.permute_dims() just like the cat function right before your code.